README.md

```diff
@@ -17,9 +17,9 @@ There are a few API endpoints we care about:
 
 ## /v2/links/fetch/`locale`
 
-The `fetch` endpoint takes locale in a JSON payload and returns an `HTTP 303` redirect to a CDN location with the content appropriate for the user agent.
+The `fetch` endpoint takes locale from the url path and returns an `HTTP 303` redirect to a CDN location with the content appropriate for the user agent.
 
-In Firefox, this is called when the `idle-daily` event is triggered. This is guaranteed to trigger only once daily.
+In Firefox, this is called when the [`idle-daily`](https://dxr.mozilla.org/mozilla-central/search?q=idle-daily&redirect=false&case=false) event is triggered. This is guaranteed to trigger only once daily.
```

Comment:
This comment is no longer true. We changed the behavior.

Maybe this:

In Firefox, the fetch endpoint is called at most once daily. Firefox keeps track of the last time it fetched, and checks at every opportunity, (e.g. when sending a ping, when a new tab page is open, on browser start) whether or not it should fetch again. This is guaranteed to trigger at most once daily.